config/cscs.py

```diff
@@ -773,6 +875,33 @@
             'cxx': 'g++',
             'ftn': 'gfortran'
         },
+        {
+            'name': 'PrgEnv-cray',
+            'target_systems': [
+                'alps', 'eiger', 'rigi', 'pilatus'
```

Comment:
I would only put the systems that we have defined here.

```suggestion
                'eiger', 'pilatus'
```